### ISSUE TITLE
ImportError: cannot import name 'patterns' with django version greater than 1.8

### ISSUE THREAD

**Issue by pdomineaux** [jgiannuzzi/taiga-contrib-saml-auth#1]
if like me you use a recent version of django > 1.8 (mine is 1.10.6)

you will have trouble with the patterns function :

```
Trying import local.py settings...
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7f959de56a60>
Traceback (most recent call last):
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/core/management/commands/runserver.py", line 113, in inner_run
    autoreload.raise_last_exception()
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/utils/autoreload.py", line 249, in raise_last_exception
    six.reraise(*_exception)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/home/taiga/taiga-back/taiga/feedback/apps.py", line 31, in ready
    from taiga.urls import urlpatterns
  File "/home/taiga/taiga-back/taiga/urls.py", line 89, in <module>
    urlpatterns += [url(r'^saml/', include('taiga_contrib_saml_auth.urls'))]
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/django/conf/urls/__init__.py", line 50, in include
    urlconf_module = import_module(urlconf_module)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/taiga/.virtualenvs/taiga/lib/python3.5/site-packages/taiga_contrib_saml_auth/urls.py", line 1, in <module>
    from django.conf.urls import patterns, url
ImportError: cannot import name 'patterns'
```

Because patterns has been deprecated.

You'll have to modify the following file like this :

file : "~/.virtualenvs/taiga/lib/python3.5/site-packages/taiga_contrib_saml_auth/urls.py"

```python
from django.conf.urls import *

from . import views

app_name = 'taiga_contrib_saml_auth'

urlpatterns = [
    url(r'^initiate-login/$', views.initiate_login, name="login_initiate"),
    url(r'^complete-login/$', views.complete_login, name="login_complete"),
    url(r'^initiate-logout/$', views.initiate_logout, name="logout_initiate"),
    url(r'^complete-logout/$', views.complete_logout, name="logout_complete"),
    url(r'^metadata/$', views.metadata, name="metadata"),
]
```

**Comment by tej941** [comment]
For url command in urlpatterns, it's showing SyntaxError: invalid syntax.
What to do?

**Comment by jgiannuzzi** [comment]
Hi @tej941, this is a closed issue. which was fixed in version 1.0.2.

Please open a new issue with more details (i.e. a stacktrace) if you have a different issue.
